collections/learn-to-code/index.md

```diff
@@ -5,12 +5,30 @@ items:
  - railsbridge/docs
  - freeCodeCamp/freeCodeCamp
  - leachim6/hello-world
+ - karan/Projects
+ - sindresorhus/awesome
+ - donnemartin/system-design-primer
```

Comment:
This looks like a great resource, but it feels more intermediate/advanced than "beginner" to me.

Reply:
Please what’s that code? I’m new here still finding my way around

Reply:
👋 Hi! The code in this pull request is a list of repositories that we show on https://github.com/collections/learn-to-code. The repository I commented on, https://github.com/donnemartin/system-design-primer, is a resource for learning about how to design large-scale systems.